code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -424,6 +424,18 @@
 	drink_desc = "Does... does this mean that Arthur and Ford are on the station? Oh joy."
 	taste_message = "the number fourty two"
 
+/datum/reagent/consumable/ethanol/flaming_homer
+	name = "Flaming Moe"
+	id = "flamingmoe"
+	description = "This seems like a bunch of various alcohols thrown together, along with what looks like prescription medicine..."
```

Comment:
Similarly `seems like` and `looks like`, grammatically I'd change the `what looks like` to `appears to be`.